sdk/docs/pages/typescript/cryptography/keypairs.mdx

```diff
@@ -125,3 +125,49 @@ const exampleMnemonic = 'result crisp session latin ...';
 
 const keyPair = Ed25519Keypair.deriveKeypair(exampleMnemonic);
 ```
+
+## Exporting and importing key pairs
+
+`Keypair`s implement an `export` function that can be used to export a keypair in a standard format.
+The return value is an object with a `scheme` property that indicates what the type of keypair was
+exported, and a `privateKey` that you can use to derive the `Keypair` again.
+
+```typescript
+import { Ed25519Keypair } from '@mysten/sui.js/keypairs/ed25519';
+import { Secp256k1Keypair } from '@mysten/sui.js/keypairs/Secp256k1';
+import { Secp256r1Keypair } from '@mysten/sui.js/keypairs/Secp256r1';
+import { Keypair, ExportedKeypair } from '@mysten/sui.js/cryptography';
+import { fromB64 } from '@mysten/sui.js/utils';
+
+const keypair = new Ed25519Keypair();
+const exported = keypair.export();
+const imported = fromExportedKeypair(exported);
+
+export function fromExportedKeypair(keypair: ExportedKeypair): Keypair {
+	const secretKey = fromB64(keypair.privateKey);
+	switch (keypair.schema) {
+		case 'ED25519':
+			return Ed25519Keypair.fromSecretKey(pureSecretKey);
+		case 'Secp256k1':
+			return Secp256k1Keypair.fromSecretKey(secretKey);
+		case 'Secp256r1':
+			return Secp256r1Keypair.fromSecretKey(secretKey);
+		default:
+			throw new Error(`Invalid keypair schema ${keypair.schema}`);
+	}
+}
+```
+
+## Deriving a `Keypair` from a hex encoded secret key
```

Comment:
Do we tell people how to encode something into hex? Otherwise this feels a little out of place to me.

Reply:
This was an ask from a wallet builder I think?  I don't remember exactly, but at least one person was specifically asking for this